cypress/support/constants.js

```diff
@@ -28,9 +28,11 @@ export const LOAN_TYPE_NAMES = {
 
 export const MATERIAL_TYPE_NAMES = {
   MICROFORM: 'microform',
+  BOOK: 'book',
 };
 
 export const CY_ENV = {
+  CHECKOUT_ITEM: 'checkoutItem',
```

Comment:
don't support CY_ENV please. I suggest to work with return value of fragments methods instead

Reply:
Done